katdal/chunkstore_s3.py

```diff
@@ -467,6 +483,10 @@ def create_array(self, array_name):
 
         if self.expiry_days > 0:
             xml_payload = _BASE_LIFECYCLE_POLICY.format(self.expiry_days)
```

Comment:
So `xml_payload` is always `_BASE_LIFECYCLE_POLICY`? When would it not validate - a bad value for `expiry_days`? It seems like a lot of effort to solve future problems.

Reply:
The grand scheme is that you will sometimes want to override the base for very user specific things - like managing storage migration say. I can see us ending up with quite a number of policies as future versions of CEPH support more elaborate management via lifecycle policies.